package.json

```diff
@@ -77,7 +78,7 @@
     "mkdirp": "1.0.4",
     "mocha": "8.3.1",
     "np": "7.4.0",
-    "openlayers": "4.6.5",
+    "ol": "git://github.com/openlayers/openlayers.git#v6.5.0",
```

Comment:
Can we not use a proper version here? Or is the npm package missing things we need?

Reply:
We need an ol legacy version which is no longer provided in the official ol package.
I opted to check out the git and build the legacy version on `postinstall`. This is not optimal as every dependant project would need to build openlayers as well. But it might be ok, what do you think?
I think the nicest way would be to provide a npm package with the built openlayers legacy version. But this would require extra work for every openlayers version update.

Reply:
or we could provide a built ol version within the geoext package.

Reply:
I think using the git repo is ok, I was just wondering why you didn't use the npm package.

Reply:
Is 6.5 really unavailable? https://www.npmjs.com/package/ol

Reply:
It is available, but we need the legacy build, which is not included in the package.

Reply:
the legacy build exports everything to an `ol` object so you can write i.e. `new ol.format.WKT`